unit10.js

```diff
@@ -0,0 +1,127 @@
+// Bai1 Phep nhan
+const calcMultipliers = n => {
+    for (let i=1; i<=10; i++) {
```

Comment:
```suggestion
  for (let i=1; i <= 10; i++) {
```
1 tab = 2 spaces
1 lần tab sai = 1 lần comment
Ngoài ra nhìn kỹ dấu cách